src/LiveApi.js

```diff
@@ -81,7 +83,7 @@ export default class LiveApi {
     }
 
     connect(connection: ?WebSocket): void {
-        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}`;
+        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}${this.brand ? `&brand=${this.brand}` : ''}`;
```

Comment:
why we need to empty parameter? its not a mandatory parameter so we should only add it as and when needed. Backend will be able to handle it if you pass it blank thats not a problem.